### DISCUSSION TITLE
Quick Steps empty export when there are quick steps

### DISCUSSION THREAD

**Discussion by breenyoung** [microsoft/mfcmapi#462] [Q&A]
Hi guys, 

Maybe you can help me with this issue I'm  having.   I have Outlook 2016 (x64) installed, with about a dozen quick steps configured.  Run MFCMAPI (x64), open a session to my only account, located root folder, quick steps (this computer only) node.  

Right click and try to export the folder as MSG files.  The output folder ends up being empty.  If I try 'View Associated Contents' on the folder similar problem in that the new window looks like I have no quick steps whatsoever configured.

If I exit out of MFCMAPI and relaunch Outlook I do have my quick steps and all are functioning properly.  What am I doing wrong?

Thanks

**Comment by stephenegriffin** [comment]
It's unclear what you're trying to do here. Why are you trying to export messages in the first place?

**Comment by breenyoung** [comment]
Hi @stephenegriffin thanks for the reply.  I'm trying to backup my quick steps using this method (IMAP):

https://www.msoutlook.info/question/backup-and-restore-quick-steps

**Comment by stephenegriffin** [comment]
Ok - so this isn't an MFCMAPI issue - it's an issue with that article and where Quick Steps are actually stored.

**Comment by breenyoung** [comment]
So where are Quick Steps actually stored?
